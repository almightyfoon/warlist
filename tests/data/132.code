c5501b3y3y3LdPdP393u393u3x3x___-0z0G0w0w3y3y3LdPdP35363u35363uhThT