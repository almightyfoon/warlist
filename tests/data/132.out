https://conflictchamber.com/#c5501b3y3y3LdPdP393u393u3x3x___-0z0G0w0w3y3y3LdPdP35363u35363uhThT

Retribution Army - 70 / 75 points

!!! This army must contain a warcaster.

Arcanist Mechanik [2]
Arcanist Mechanik [2]
Fane Knight Skeryth Issyen [8]
Houseguard Thane [4]
Houseguard Thane [4]
Houseguard Riflemen (max) [15]
 - Soulless Escort (1) [1]
Houseguard Riflemen (max) [15]
 - Soulless Escort (1) [1]
Stormfall Archers [9]
Stormfall Archers [9]

Retribution Army - 75 / 75 points
[Theme] Defenders of Ios

[Kaelyssa 1] Kaelyssa, the Night's Whisper [+29]
 - Hydra [15]
 - Hydra [15]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Fane Knight Skeryth Issyen [0(8)]
Houseguard Thane [0(4)]
Houseguard Thane [0(4)]
Houseguard Halberdiers (max) [13]
 - Houseguard Halberdier Officer & Standard [4]
 - Soulless Escort (1) [1]
Houseguard Halberdiers (max) [13]
 - Houseguard Halberdier Officer & Standard [4]
 - Soulless Escort (1) [1]
Arcantrik Force Generator [17]
Arcantrik Force Generator [17]
